### PR TITLE
fix: use minimal image when deploying nfd chart

### PR DESCRIPTION
**Pull request by faiq**
**What problem does this PR solve?**:
When deploying with the helm addon strategy i noticed that we did not deploy the minimal image. 

I'm going to follow this up with a PR that automates syncing the values in the templates and `hack/addons/kustomize/**/helm-values.yaml` 

**Which issue(s) this PR fixes**:
Fixes #

**How Has This Been Tested?**:

**Special notes for your reviewer**:

### CODE REVIEW

File: charts/cluster-api-runtime-extensions-nutanix/templates/nfd/manifests/helm-addon-installation.yaml [hunk 1]
```diff
@@ -8,6 +8,8 @@ metadata:
   name: '{{ .Values.hooks.nfd.helmAddonStrategy.defaultValueTemplateConfigMap.name }}'
 data:
   values.yaml: |-
+    image:
+      tag: v0.15.2-minimal
```
Comment (dkoshkin) [inline charts/cluster-api-runtime-extensions-nutanix/templates/nfd/manifests/helm-addon-installation.yaml:12]: Do we get a variable with the chart version? Or can automatically set this when generating these values?
We will forget to change this.

Comment (faiq) [reply charts/cluster-api-runtime-extensions-nutanix/templates/nfd/manifests/helm-addon-installation.yaml:12]: i'll be following up with a tool to sync these files with. this was just one i noticed that was broken right now 

hack/addons/kustomize/**/helm-values.yaml

**Review by dkoshkin — APPROVED** [review]

**Comment by jimmidyson** [comment]
Fixing the `lint-test-helm` check in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/779.

File: charts/cluster-api-runtime-extensions-nutanix/templates/nfd/manifests/helm-addon-installation.yaml (hunk 1 above)
Comment (jimmidyson) [reply charts/cluster-api-runtime-extensions-nutanix/templates/nfd/manifests/helm-addon-installation.yaml:12]: Merge this and follow up in a different PR, i guess that's what you're saying @faiq ?

**Review by jimmidyson — APPROVED** [review]

**Pull request merged**